doc/tutorial/index.rst

```diff
@@ -0,0 +1,198 @@
+.. _tutorial:
+
+===============
+Sphinx tutorial
+===============
+
+In this tutorial you will build a simple documentation project using Sphinx,
+and view it in your browser as HTML.
+The project will include narrative, handwritten documentation,
+as well as autogenerated API documentation.
+
+The tutorial is aimed towards Sphinx newcomers
+willing to learn the fundamentals of how projects are created and structured.
+You will create a fictional software library to generate random food recipes
+that will serve as a guide throughout the process,
+with the objective of properly documenting it.
+
+To showcase Sphinx automatic documentation generation capabilities
+you will use Python, which is the default :term:`domain`
+(even though several other languages are supported,
+they all work in a similar way).
+
+To follow the instructions you will need access to a Linux-like command line
+and a basic understanding of how it works,
+as well as a working Python installation for development,
+since you will use *Python virtual environments* to create the project
+(you can read more about them in the `Python Packaging User Guide`_).
+
+.. _Python Packaging User Guide: https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment
+
+Getting started
+---------------
+
+Setting up our project and development environment
+..................................................
+
+On a new directory,
+create a file called ``README.rst``
+with the following contents:
+
+.. code-block:: rest
+
+   Lumache
+   =======
+
+   **Lumache** (/luˈmake/) is a Python library for cooks and food lovers
+   that creates recipes mixing random ingredients.
+
+It is a good moment to create a Python virtual environment
+and install the required tools.
+For that, open a command line terminal,
+``cd`` into the directory you just created,
+and run the following commands:
+
+.. code-block:: bash
+
+   $ python -m venv .venv
```

Comment:
Our installation document does not require to install the venv library. So it might not be available in some Linux distro. For example, it is failed in Ubuntu 20.04:
```
root@b5f8ca310971:/# python3 -m venv .venv
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/.venv/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']
```

Reply:
You're right, this was a concern for me as well. This is a very Ubuntu and Debian-specific problem, but a very prevalent and relevant one. I'd rather keep using virtual environments though. Do you think we should expand `installation.rst` to account for this?

Reply:
IMO, it's important for beginners not familiar with python especially. It's needed to mention in installation.rst or here. It's not a "required" package, but a "recommended" one for the tutorial.

Reply:
I think it makes more sense in `installation.rst`. I'll update it on this PR if you don't mind.

Reply:
+1

Reply:
Done!